frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -107,136 +121,145 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
   }
 
   return (
-    <Stack hasGutter style={{ minHeight: 400 }}>
-      <StackItem>
-        <LogsTabStatus
-          loaded={loaded}
-          error={error}
-          refresh={refreshLogs}
-          onDownload={onDownloadAll}
-        />
-      </StackItem>
-      <StackItem>
-        <Split hasGutter isWrappable>
-          <SplitItem>
-            <Split hasGutter>
-              {isSmallScreen() ? null : (
-                <SplitItem>
-                  <Bullseye>
-                    <TextContent>
-                      <TextListItem component={TextListItemVariants.dt}>Step</TextListItem>
-                    </TextContent>
-                  </Bullseye>
-                </SplitItem>
-              )}
+    <DashboardLogViewer
+      data={data}
+      toolbar={
```

Comment:
Is it okay to pass the toolbar as a prop or should I move this to DashboardLogViewer.tsx file?

Reply:
It's fine to pass it as a prop -- the concern I have here is your prop in itself is 90% of the file lol. We may want to consider (not with this PR) breaking this down and properly getting a nice breakup of components so this is easier to understand and maintain.